fastai/vision/augment.py

```diff
@@ -54,29 +69,42 @@ def flip_lr(x:TensorBBox):  return TensorBBox(TensorPoint(x.view(-1,2)).flip_lr(
 # Cell
 class FlipItem(RandTransform):
     "Randomly flip with probability `p`"
-    def __init__(self, p=0.5): super().__init__(p=p)
+    def __init__(self, p:float=0.5): super().__init__(p=p)
     def encodes(self, x:(Image.Image,*TensorTypes)): return x.flip_lr()
 
 # Internal Cell
 @patch
-def dihedral(x:PILImage, k): return x if k==0 else x.transpose(k-1)
+def dihedral(
+    x:PILImage,
+    k:int, # Order of Dihedral Transformation from 0 to 7
+):
+    return x if k==0 else x.transpose(k-1)
 @patch
-def dihedral(x:TensorImage, k):
-        if k in [1,3,4,7]: x = x.flip(-1)
-        if k in [2,4,5,7]: x = x.flip(-2)
-        if k in [3,5,6,7]: x = x.transpose(-1,-2)
-        return x
+def dihedral(
+    x:TensorImage,
+    k:int, # Order of Dihedral Transformation from 0 to 7
```

Comment:
I believe this is a better description
```suggestion
    k:int, # Dihedral transformation to apply
```